reference/price_lists.v3.yml

```diff
@@ -3885,205 +3848,72 @@ components:
         type: object
       title: Price Record Collection Put
       x-internal: false
+    PriceRecordCollectionPutWithPriceListId:
+      type: array
+      items:
+        $ref: "#/components/schemas/PriceRecordBatchItem"
     PriceRecordBatchItem:
-      description: Price Record object used in batch create or update request.
+      type: object
+      description: Price Record object used in batch create or update.
       allOf:
-        - properties:
+        - $ref: "#/components/schemas/PriceRecordBase"
+        - type: object
+          properties:
+            price_list_id:
+              type: integer
+              description: The variant with which this price record is associated. Either `variant_id` or `sku` is required.
```

Comment:
check the descriptions through here. They appear to be copy-paste leftovers or TO-DOs.